registration-processor/pre-processor/registration-processor-supervisor-validator-stage/src/test/java/io/mosip/registration/processor/app/SupervisorValidatorProcessorTest.java

```diff
@@ -151,8 +152,9 @@ public void setUp() throws Exception {
 	public void testisValidOperatorSuccess() throws Exception {
```

Comment:
Why is this method called OperatorSuccess ?

Reply:
changed this method name...